etc/pyfstat-dev.yml

```diff
@@ -15,8 +15,8 @@ dependencies:
   - peakutils
   - pathos
   - python-lal >=7.1.5
-  - python-lalpulsar >=3.1.1
-  - lalapps >=7.4.0
+  - python-lalpulsar >=5.0.0
+  - lalpulsar >=5.0.0
```

Comment:
Exactly. https://anaconda.org/conda-forge/lalpulsar
->
> This package contains the shared-object libraries needed to run applications that use the LAL Pulsar library. If you want to install the Python bindings, please install the associated python-lalpulsar package.

And now it also includes the executables, which is why we need it now instead of lalapps.